.github/ISSUE_TEMPLATE/rule-blocking-bug.md

```diff
@@ -12,8 +12,8 @@ assignees: ''
 - Fill in the following information
 
 **Links to related JIRA Tickets**
-- https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-
```

Comment:
I don't think these should be updated. I believe the links weren't working before

Reply:
I understand, that's somehow got there by branch merging, reverted